### ISSUE TITLE
如何进行多标签分类，n_class设置不起作用

### ISSUE THREAD

**Issue by 975150313** [junqiangchen/HC18-Automated-measurement-of-fetal-head-circumference#5]
请问一下如何进行多标签分类，n_class设置不起作用 ，logits，labels怎么处理以适应多标签分类？

**Comment by junqiangchen** [comment]
你好，这是两类分割，如果想要多分类的话，可以参考这个项目[https://github.com/junqiangchen/BraTS18-Challege](url)

**Comment by 975150313** [comment]
> 你好，这是两类分割，如果想要多分类的话，可以参考这个项目[https://github.com/junqiangchen/BraTS18-Challege](url)

感谢您的回复，我查阅资料了解到，通过修改损失函数，我已经实现了，多标签分类和二分类损失函数是相同的，我把损失函数修改为binary_crossentropy ，类别数量改成我自己的就可以
